cmd/podman/pod_stop.go

```diff
@@ -46,12 +48,41 @@ func init() {
 }
 
 func podStopCmd(c *cliconfig.PodStopValues) error {
+	if os.Geteuid() != 0 {
+		rootless.SetSkipStorageSetup(true)
+	}
+
 	runtime, err := adapter.GetRuntime(&c.PodmanCommand)
 	if err != nil {
 		return errors.Wrapf(err, "could not get runtime")
 	}
 	defer runtime.Shutdown(false)
 
+	if rootless.IsRootless() {
```

Comment:
This appears to be the same code as you did in rm.  If you think you'd need it elsewhere eventually, I'd make it a function in common.  If not I'd still rather see it there, but not a biggy.

Reply:
good idea.  Refactored in a function